assets/component-pickup-availability.css

```diff
@@ -42,7 +42,7 @@ pickup-availability[available] {
 
 pickup-availability-drawer {
   background-color: rgb(var(--color-background));
-  border: 0.1rem solid rgba(var(--color-foreground), 0.2);
+  border: var(--popup-drawer-border-width) solid rgba(var(--color-foreground), var(--popup-drawer-border-opacity));
```

Comment:
Should there be a border to the right ? 
![](https://screenshot.click/21-11-9v702-0knx6.png)

Reply:
Just do border left! 👍